pkg/util/webhookpatch.go

```diff
@@ -98,6 +99,9 @@ func PatchValidatingWebhookConfig(client admissionregistrationv1beta1client.Vali
 	if err != nil {
 		return err
 	}
-	_, err = client.Patch(webhookConfigName, types.StrategicMergePatchType, patch)
+
+	if string(patch) != "{}" {
+		_, err = client.Patch(webhookConfigName, types.StrategicMergePatchType, patch)
+	}
```

Comment:
Lines 103-105 are correct for validating webhook. You need to make the same change for the mutating webhook on line 61.

Reply:
Done